src/components/OpeningTimes.js

```diff
@@ -24,11 +24,11 @@ class OpeningTimes extends React.Component {
 
         var todayOpen = object.opening_hours[
             moment().day()
-        ];
+        ] || {};
 
         var nextOpen = object.opening_hours[
             (moment().day() + 1) % object.opening_hours.length
-        ];
+        ] || {};
```

Comment:
Yeah. I'm actually not 100% sure I'm doing this right anyway. It might be we need to use `_.findWhere` to find the right day.